examples/api/plot_softplus.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+"""
+===============================
+Poisson (Softplus) Distribution
+===============================
+
+For Poisson distributed target variables, the canonical link function is an
+exponential nonlinearity. However, the gradient of the loss function that uses
+this nonlinearity is typically unstable.
+
+This is an example demonstrating how pyglmnet
+works with a Poisson distribution using the softplus nonlinearity.
+
+Here, for the ``distr = 'softplus'`` option, we use the
+softplus function: ``log(1+exp())``.
+
+"""
```

Comment:
actually, it just occured to me. Can we call the folder the example is in something other than API? It's a bit confusing to me ...